server/src/weaverbird/backends/mongo_translator/steps/ifthenelse.py

```diff
@@ -22,7 +22,7 @@ def translate_ifthenelse(step: IfthenelseStep) -> list[MongoStep]:
         {
             '$addFields': {
                 step.new_column: transform_ifthenelse_step(
-                    IfThenElse(condition=step.condition, then=step.then, else_value=step.else_value)
+                    IfThenElse(condition=step.condition, then=step.then, else_value=step.else_value)  # type: ignore
```

Comment:
To be honnest, i don't know why the linter is not happy here... WEIRD 🤔

Reply:
pydantic plugin doesn't support well `allow_population_by_field_name`. So it looks for aliases which are `if` and `else` instead of `condition` and `else_value`